cropduster/models.py

```diff
@@ -522,7 +522,7 @@ def _save_standalone_thumb(self, size, image=None, thumb=None, commit=True):
         return thumb
 
     def _save_thumb(self, size, image=None, thumb=None, ref_thumb=None, tmp=False, commit=True):
-        image = image or PIL.Image.open(safe_str_path(self.image.path))
+        image = image or PIL.Image.open(safe_str_path(self.image.name))
```

Comment:
Related to my other comment about `PIL.Image.open`, in order to actually use the right storage for s3 you'll need to change this to something like:

```suggestion
        if not image:
            with self.image as f:
                f.open()
                image = PIL.Image.open(BytesIO(f.read())
```